### PR TITLE
Add extra SUP fields

### PR DESCRIPTION
**Pull request by maxwofford**
WIP-- this is a bundle of changes around givebutter signup & admin tools that @hackliv requested

### CODE REVIEW

**Comment by exu3** [comment]
@maxwofford What still needs to be done here to merge / anything I could do to help?

**Comment by maxwofford** [comment]
I had a chat with @hackliv & @MelSmith104 and they had a couple things they wanted changed. This PR should address these things:

**Application flow:**
- [ ] SUP should have an admin show page - https://github.com/hackclub/bank/pull/2206/commits/90811ffb94b4e86f24252136231d8573d1a03bc8
- [ ] Accept and reject buttons should give an email template to reach out with them (bcc bank@)
	- [ ] Approve has “calendly link & name”
	- [ ] Reject had just name
- [x] Duplicate signup data to Airtable - #2330 
- [ ] Events admin index should have partnered orgs and a dropdown 
- [ ] Ledger should have a filter by partner

**Misc:**
- [x] Events search page shows the same count of events for the past couple months - #2292 
- [ ] Need to add partner filter for all the admin search pages

**Comment by garyhtou** [comment]
@eilla1 let's work on this during our next pair programming call!

cc @maxwofford let's aim to have the state machine done before then so that it can be used here

File: app/models/partnered_signup.rb [hunk 1]
```diff
@@ -66,6 +69,30 @@ class PartneredSignup < ApplicationRecord
...
 
+  def admin_reject_redirect_url
+    recipient = "#{owner_name} <#{owner_email}>"
+    bcc = "bank-outgoing@hackclub.com"
+    subject = "Application to Hack Club Bank"
+    reason = ""
+    "mailto:#{CGI.escape(recipient)}?bcc=#{bcc}&subject=#{CGI.escape(subject)}&body=#{CGI.escape(reason)}"
+
+    "mailto:gary@hackclub.com"
+  end
```
Comment (exu3) [inline app/models/partnered_signup.rb]: `mailto:` opens the email template in a user's default email client (Mail.app for mac); is that desired behavior?

Would something like https://mail.google.com/mail/?fs=1&su=Hack+Club+Bank&body=hi+there&to=ella@hackclub.com&bcc=bank-outgoing@hackclub.com&tf=cm work better cause it opens it in a Gmail window which I'm assuming is where people would send emails from? (I'm not sure; I could be wrong here)

Comment (garyhtou) [reply app/models/partnered_signup.rb]: good point! i think that's a good idea since Liv will prob be using gmail

**Comment by maxwofford** [comment]
Closing this b/c the givebutter api project is stale at time of writing.

**Pull request closed without merging**